### DISCUSSION TITLE
I can't upgrade to tailwind v3 in react

### DISCUSSION THREAD

**Discussion by sachinit254** [tailwindlabs/tailwindcss#6338] [Help]
Here is my package.json  

 {
  "name": "book-app",
  "version": "0.1.0",
  "private": true,
  "proxy": "http://127.0.0.1:5000/",
  "dependencies": {
    "@emotion/react": "^11.4.1",
    "@emotion/styled": "^11.3.0",
    "@fortawesome/fontawesome-svg-core": "^1.2.36",
    "@fortawesome/free-solid-svg-icons": "^5.15.4",
    "@fortawesome/react-fontawesome": "^0.1.16",
    "@material-ui/core": "^4.12.3",
    "@material-ui/icons": "^4.11.2",
    "@material-ui/lab": "^4.0.0-alpha.60",
    "@mui/icons-material": "^5.0.1",
    "@mui/material": "^5.0.2",
    "@mui/styles": "^5.0.1",
    "@testing-library/jest-dom": "^5.14.1",
    "@testing-library/react": "^11.2.7",
    "@testing-library/user-event": "^12.8.3",
    "and": "0.0.3",
    "axios": "^0.21.4",
    "postcss-cli": "^8.3.1",
    "react": "^17.0.2",
    "react-dom": "^17.0.2",
    "react-redux": "^7.2.5",
    "react-router-dom": "^5.2.0",
    "react-scripts": "4.0.3",
    "redux": "^4.1.1",
    "redux-devtools-extension": "^2.13.9",
    "redux-thunk": "^2.3.0",
    "web-vitals": "^1.1.2"
  },
  "scripts": {
    "start": "react-scripts start",
    "build": "react-scripts build",
    "test": "react-scripts test",
    "eject": "react-scripts eject"
  },
  "eslintConfig": {
    "extends": [
      "react-app",
      "react-app/jest"
    ]
  },
  "browserslist": {
    "production": [
      ">0.2%",
      "not dead",
      "not op_mini all"
    ],
    "development": [
      "last 1 chrome version",
      "last 1 firefox version",
      "last 1 safari version"
    ]
  },
  "devDependencies": {
    "autoprefixer": "^10.4.0",
    "postcss": "^8.4.4",
    "tailwindcss": "^3.0.0"
  }
}

**Comment by HNazmul-X** [comment]
Hey, for using Tailwind CSS in with react you have to use tailwind. config.js and postcss.js or craco. config.js

**Reply by sachinit254** [reply]
I have these files
